### DISCUSSION TITLE
Last year's news (outdated neovim.io frontpage)

### DISCUSSION THREAD

**Discussion by morzel85** [neovim/neovim#24342] [General]
What's up with the "News" section on [neovim.io](neovim.io)? It gives a dead project vibes.
Notice that last version is from 2022.4, preceded (sadly) by Christmas 2021 issue... and Twitter feed has nothing (login requirement?).

![image](https://github.com/neovim/neovim/assets/7968512/81d3b63d-67df-4572-9435-8adbbfd03605)

**Comment by clason** [comment]
Twitter is... Twitter; no need to comment on what is over all the news.

And we replaced the newsletters on the website with much more useful `:h news` running release notes.

That being said, if you have concrete, actionable suggestions for improvement, we'd be happy to hear them.

This issue isn't.

**Reply by morzel85** [reply]
If the news section is not maintained it would be best to removed it or provide a feed from GitHub releases.

**Reply by clason** [reply]
PR welcome. (Adding news, not removing content.)
